config/src/test/java/beanconfig/ArraysConfig.java

```diff
@@ -11,6 +11,7 @@
 public class ArraysConfig {
 
     List<Integer> empty;
+    List<Integer> fromEmptyString;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/hp/config/file/2444096851/issues/source?bid=3245965&fileBranchId=3245971#l14)